omero/conf_autogen.py

```diff
@@ -13,6 +13,6 @@
 version_dsl_plugin = "5.5.0"
 version_blitz_plugin = "5.5.0"
 version_ice_builder = "1.5.0"
-version_py = "5.5.dev2"
```

Comment:
This is handled via the autogen step generally, but bumping manually doesn't hurt. :+1: